jumeg_causality/dirs_manage.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
body { width: 100% !important; -webkit-text-size-adjust: 100%; -ms-text-size-adjust: 100%; margin: 0; padding: 0; }
      img { outline: none; text-decoration: none; -ms-interpolation-mode: bicubic; }
      body { color: #65646a; font-family: 'Helvetica Neue', HelveticaNeue, helvetica, arial, sans-serif; }
      h1 a:active { color: #4a8db8 !important; }
      h2 a:active { color: #4a8db8 !important; }
      h3 a:active { color: #4a8db8 !important; }
      h4 a:active { color: #4a8db8 !important; }
      h5 a:active { color: #4a8db8 !important; }
      h6 a:active { color: #4a8db8 !important; }
      h1 a:visited { color: #4a8db8 !important; }
      h2 a:visited { color: #4a8db8 !important; }
      h3 a:visited { color: #4a8db8 !important; }
      h4 a:visited { color: #4a8db8 !important; }
      h5 a:visited { color: #4a8db8 !important; }
      h6 a:visited { color: #4a8db8 !important; }
      a:visited { color: #4a8db8; }
      a:hover { color: #2dbe60; text-decoration: underline; }
      .u-headingMobileAnchor a {text-decoration: none !important; color: #2dbe60 !important; pointer-events: none !important; cursor: default !important;}
      .u-bodyMobileAnchor a {text-decoration: none !important; color: #65646a !important; pointer-events: none !important; cursor: default !important;}
      .footer a:hover { text-decoration: none; }
      .footer-small a:hover { text-decoration: underline; }
      body { background-color: #f3f3f3; }
      .ExternalClass { width: 100%; }
      .ExternalClass { line-height: 100%; }
      table[id='inception'] .Btn-anchor:hover { color: #fff !important; }
      table[id='inception'] .Btn-anchor:visited { color: #fff !important; }
      @media screen and (min-width: 579px) {
        table[id="inception"] .l-inception-point { width: 578px !important; }
      }
      @media only screen and (min-device-width: 768px) and (max-device-width: 1024px) {
        table[id="inception"] a[href^="tel"] { text-decoration: none; color: #65646a; pointer-events: none; cursor: default; }
        table[id="inception"] a[href^="sms"] { text-decoration: none; color: #65646a; pointer-events: none; cursor: default; }
        .mobile-link table[id="inception"] a[href^="tel"] { text-decoration: default; color: #65646a !important; pointer-events: auto; cursor: default; }
        .mobile-link table[id="inception"] a[href^="sms"] { text-decoration: default; color: #65646a !important; pointer-events: auto; cursor: default; }
        table[id="inception"] .u-headingMobileAnchor a {text-decoration: none !important; color: #2dbe60 !important; pointer-events: none !important; cursor: default !important;}
        table[id="inception"] .u-bodyMobileAnchor a {text-decoration: none !important; color: #65646a !important; pointer-events: none !important; cursor: default !important;}
      }
      @media only screen and (max-device-width: 578px), only screen and (max-width: 578px) {
        table[id="inception"] .Branding_positive-container { padding: 14px 0 0 16px !important; }
        table[id='inception'] .Btn-anchor { display: block !important; }
        table[id='inception'] .Row_small { padding: 0 !important; }
        table[id='inception'] .Row_medium-top { padding-top: 16px !important; }
        table[id='inception'] .Avatar-img { width: 48px !important; height: 48px !important; }
        table[id="inception"] { width: 100% !important; }
        table[id="inception"] .l-outie { padding: 0px !important; }
        table[id="inception"] .l-inception-point { width: 100% !important; }
        table[id="inception"] .l-footer { width: 100% !important; }
        table[id="inception"] .l-wrap { width: 100% !important; }
        table[id="inception"] .l-full-bleed { width: 100% !important; }
        table[id="inception"] .media-obj { width: 100% !important; }
        table[id="inception"] .media-content { width: 100% !important; }
        table[id="inception"] .btn { width: 100% !important; }
        table[id="inception"] .btn td { width: 100% !important; }
        table[id="inception"] .l-inception-point { border-right: none !important; border-left: none !important; }
        table[id="inception"] .l-full-bleed img { width: 100% !important; height: auto !important; }
        table[id="inception"] .l-pad > tbody > tr > td { padding: 16px !important; }
        table[id="inception"] .l-split { width: 100% !important; }
        table[id="inception"] .l-split + .l-split { margin-top: 32px !important; }
        table[id="inception"] .ill-notebook { width: 100% !important; height: 96px !important; }
        table[id="inception"] .footer { padding-right: 16px !important; padding-left: 16px !important; }
        table[id="inception"] .btn span { display: block !important; }
        table[id="inception"] a[href^="tel"] { text-decoration: none; color: #65646a; pointer-events: none; cursor: default; }
        table[id="inception"] a[href^="sms"] { text-decoration: none; color: #65646a; pointer-events: none; cursor: default; }
        .mobile-link table[id="inception"] a[href^="tel"] { text-decoration: default; color: #4a8db8 !important; pointer-events: auto; cursor: default; }
        .mobile-link table[id="inception"] a[href^="sms"] { text-decoration: default; color: #4a8db8 !important; pointer-events: auto; cursor: default; }
        table[id="inception"] .u-headingMobileAnchor a {text-decoration: none !important; color: #2dbe60 !important; pointer-events: none !important; cursor: default !important;}
        table[id="inception"] .u-bodyMobileAnchor a {text-decoration: none !important; color: #65646a !important; pointer-events: none !important; cursor: default !important;}
        table[id="inception"] .Rule { display: none !important; }
        table[id="inception"] .l-pad { text-align: center !important; }
      }
    
  
  
    
      
        
          
            
              
                
                  
                    
                      
                        
                          
                          
                            
                              
                                
                                  
                                
                              
                            
                          
                        
                      
                    
                  
                  
                  
                    
                      
                        
                          
  
    
      

        
          
            
              

              
            

            
              
                
                  Dong Qunxi shared a note with you.
                
              
            

            
              
            
              View note in Evernote
        
              
            
          
        
      
    
  



  
    
      
        Write notes, collect ideas, and move your projects forward with Evernote.
      
    
  


                        
                      
                    
                  
                  
                
                
              
            
          
          
          
            
              
                For support requests, please contact us by going to our support page.

                This email has been sent to you by Evernote Corporation (or its subsidiary). Evernote Corporation, 305 Walnut Street, Redwood City, CA 94063, USA.

                
                  
                    Unsubscribe